frontend/src/components/DragAndDrop/DragAndDrop.tsx

```diff
@@ -17,7 +17,7 @@ export const DragAndDrop = () => {
     setLines(null);
     setDragActive(false);
     setInvalidFile(false);
-  }, [location]);
```

Comment:
I'm not following - is it related to the PR topic? In any case could you explain what it does?

Reply:
Yeah actually it isn't related to this PR. During the PR #849  I had done this fix, although it worked but later I realized in the useEffect dependency I had directly passed `location` which is a bad practice because its an object. It should have been `location.key`. This was just a 1 one line fix so I included it over here.

Reply:
Makes sense but they actually still use it with object in their docs :): 
![image](https://github.com/bowtie-json-schema/bowtie/assets/26698228/64744490-3297-40fd-a96c-b91affc1a82c)

But it's fine, current solution looks more reasonable to me

Reply:
Okay then I will remove this!

Reply:
Okay!